runtime/queries/column_timeseries.go

```diff
@@ -427,7 +425,8 @@ func getFilterFromDimensionValuesFilter(
 				if lv.GetKind() == nil {
 					continue
 				}
-				likeClause += escapedName + " " + prefix + " ILIKE '" + escapeSingleQuotes(lv.GetStringValue()) + "'"
+				likeClause += escapedName + " " + prefix + " ILIKE ?"
+				args = append(args, lv.GetStringValue())
```

Comment:
We want to enforce that `lv` is a string – maybe we should change the proto to use `string` for the `like` field?